modules/bbp-experiment/src/test/resources/data/bbp/experiment/patchedslice/v0.1.0/min-fields.json

```diff
@@ -0,0 +1,24 @@
+{
+    "@context": [
+        "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
```

Comment:
"{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0" is a context for the schemas not for instances.